src/Features/Core/Portable/AddParameter/AbstractAddParameterCodeFixProvider.cs

```diff
@@ -111,74 +154,164 @@ internal abstract class AbstractAddParameterCodeFixProvider<
             }
 
             var arguments = (SeparatedSyntaxList<TArgumentSyntax>)syntaxFacts.GetArgumentsOfObjectCreationExpression(objectCreation);
+            var methodCandidates = type.InstanceConstructors;
+            var typeArguments = (SeparatedSyntaxList<TTypeSyntax>)syntaxFacts.GetTypeArgumentsOfObjectCreationExpression(objectCreation);
+            var candidates = GetArgumentInsertPositionAndTypeArgumentForMethodCandidates(semanticModel, syntaxFacts, methodCandidates, arguments, argumentOpt, typeArguments);
+            RegisterFixForMethodOverloads(context, arguments, candidates);
```

Comment:
'candidates' is a strange name here.and inconsistent with the method about where you call it argumentInsertPositionInMethodCandidates